### PR TITLE
Implemented: computation jobs section on catalog page (#85zt0m698)

### LINKED ISSUE #118: Create find screen for pre-order catalog
## What is the motivation for adding/enhancing this feature?
Merchandisers require the ability to audit whether a product is accepting pre-order/backorder, and should be able to search for a specific product directly from the Catalog page in order to view its current status.

## What are the acceptance criteria?
1. The search functionality should allow merchandisers to search for products by their SKU.
2. The product page should display the current pre-order/backorder status of the product, including any relevant details such as `from` and `through` dates.

## Can you complete this feature request by yourself?

- [ ] YES
- NO

## Additional information
Link to figma file: https://www.figma.com/file/wK5FMs4z2zXQR9VfHV9ycE/Dhiraj's-Ionic-6-Material-UI-Kit-(Community)?node-id=13416%3A0&t=DXQd9NgF4cyTehn7-1

Image for reference: 
Find Product page

### PR DESCRIPTION
**Pull request by k2maan**
### Related Issues

Closes #118 

### Screenshots of Visual Changes before/after (If There Are Any)

computation job

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/views/catalog.vue [hunk 1]
```diff
@@ -40,8 +61,8 @@
             </ion-label>
           </ion-item>
 
-          <ion-chip class="tablet" outline>
-            <ion-label>{{ product.prodCatalogCategoryTypeIds.includes('PCCT_PREORDR') ? $t('Pre-order') : $t('Back-order') }}</ion-label>
+          <ion-chip v-if="product.prodCatalogCategoryTypeIds.includes('PCCT_PREORDR') || product.prodCatalogCategoryTypeIds.includes('PCCT_PREORDR')" class="tablet" outline>
```
Comment (ymaheshwari1) [inline src/views/catalog.vue:64]: Update type for back-order

File: src/views/catalog.vue [hunk 2]
```diff
@@ -191,9 +223,58 @@ export default defineComponent({
...
+        }
+
+        // fetching last run time
+        resp = await JobService.fetchJobInformation({
+          "inputFields": {
+            ...params.inputFields,
+            "statusId": ["SERVICE_CANCELLED", "SERVICE_CRASHED", "SERVICE_FAILED", "SERVICE_FINISHED"],
+            "statusId_op": "in",
+          },
+          // fetching statusId as well as only one fiel cannot be sent
```
Comment (ymaheshwari1) [inline src/views/catalog.vue:256]:
```suggestion
          // fetching statusId as well as only one field cannot be sent
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/catalog.vue (hunk 1 above)
Comment (k2maan) [reply src/views/catalog.vue]: Fixed sir

File: src/views/catalog.vue [hunk 3]
```diff
@@ -191,9 +220,64 @@ export default defineComponent({
...
+          })
+
+          if (!hasError(resp)) {
+            this.preordBckordComputationJob.lastRunTime = resp.data.docs[0].runTime
+          }
+        } else {
+          this.preordBckordComputationJob = {}
+        }
+      } catch (error) {
+        showToast(translate("Something went wrong"));
```
[the remark this answers is not in the document]
Comment (k2maan) [reply src/views/catalog.vue]: Skipped sir

File: src/views/catalog.vue [hunk 4]
```diff
@@ -191,9 +223,58 @@ export default defineComponent({
...
+      try {
+        const params = {
+          "inputFields": {
+            "statusId": "SERVICE_PENDING",
+            "statusId_op": "equals",
+            "shopId_fld0_value": this.currentEComStore.productStoreId,
+            "shopId_fld0_grp": "1",
+            "shopId_fld0_op": "equals",
+            "shopId_fld1_grp": "2",
+            "shopId_fld1_op": "empty",
```
[the remark this answers is not in the document]
Comment (k2maan) [reply src/views/catalog.vue]: Updated

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**